src/lib/app.js

```diff
@@ -55,6 +56,16 @@ class App {
     koaApp.ws.use(passport.initialize())
     koaApp.ws.use(websocketRouter.routes())
     koaApp.ws.use(websocketRouter.allowedMethods())
+    // Override listen() in order to specify a maxPayload. See:
+    //   https://github.com/kudos/koa-websocket/blob/785001c20d94fbb60cb05596bb1869889f539351/index.js#L15-L20
+    // for the original version.
+    koaApp.ws.listen = function (server) {
```

Comment:
If you need to overwrite one of 3 methods provided by this library is it even worth using the library? If we do stick with this we should submit a PR or file an issue on the module's repo about allowing you to specify additional options.